features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/utils/ReflectionUtils.kt

```diff
@@ -97,4 +112,39 @@ internal class ReflectionUtils {
     fun getClipShape(modifier: Modifier): Shape? {
         return ComposeReflection.ClipShapeField?.getSafe(modifier) as? Shape
     }
+
+    fun getBitmapInVectorPainter(vectorPainter: VectorPainter): Bitmap? {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ Do you have tests for these functions that you moved here ?

Reply:
No, reflection functions are tricky to test, they heavily depend on runtime environment. That's why I move all the reflection functions into this single class, and we are able to mock it and ensure the other parts by unit test.